src/test/java/com/kdt/lecture/repository/CustomerRepositoryTest.java

```diff
@@ -0,0 +1,134 @@
+package com.kdt.lecture.repository;
+
+import com.kdt.lecture.domain.Customer;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Slf4j
+@SpringBootTest
```

Comment:
```suggestion
@DataJpaTest
```

Repository를 대상으로 하는 테스트의 경우 @DataJpaTest를 사용하시는것이 좋을 것 같습니다.